### ISSUE TITLE
Feeds load on every keystroke when using a step template

### ISSUE THREAD

**Issue by droyad** [OctopusDeploy/Issues#6735]
## Steps to reproduce
1. Create a step template based on the `Script` step and use an inline script (don't specify any packages)
2. Use that template in a deployment process
3. Observe that the `Feed` API is called on every keystroke (the `Save` button may also flicker)

## Cause
Suspect a component is being removed and re-added and that component loads feeds every time

![SaveButtonMaddness](https://user-images.githubusercontent.com/1687639/104552267-359bb200-5684-11eb-9802-990b00166f63.gif)

**Comment by droyad** [comment]
It is [this line](https://github.com/OctopusDeploy/OctopusDeploy/blob/59a3addde33b02a2f07c91c55f9a58bdb3c7c823/newportal/app/components/ActionTemplateEditor/ActionTemplateEditor.tsx#L108) in `ActionTemplateEditor` which is causing the [feeds to load](https://github.com/OctopusDeploy/OctopusDeploy/blob/59a3addde33b02a2f07c91c55f9a58bdb3c7c823/newportal/app/components/ActionTemplateEditor/ActionTemplateEditor.tsx#L123)

**Comment by MarkSiedle** [comment]
Fixed in [this commit](https://github.com/OctopusDeploy/OctopusDeploy/commit/096ffbade0f20f765eb310afde6aef845702ba35). I deemed this not large enough for a PR. The offending logic was clear (re-fetching on every update). The component was correctly being mounted on every step-change, so this additional fetch was unnecessary and overkill. I checked related areas of the process editor's code (going back to the original commit that caused the problem) and this was the only instance I could find. This removal feels very safe.
